webui/tests/test-cluster/Makefile

```diff
@@ -33,3 +33,16 @@ prep-root:
 .PHONY: run
 run: prep-root
 	@python3 run-server.py
+
+.PHONY: fmt
+fmt:
+	isort -y
+	black .
+
+.PHONY: check
+check:
+	isort --check-only
+	black . --check
+	flake8
+	mypy . scripts
+	
```

Comment:
nit: new line at the end? or keep it as is if it's matching the tools dir.

Reply:
> LGTM.
> is [5332b7a](https://github.com/determined-ai/determined/commit/5332b7a317448792d94aaa16e5830d5178cd33d6) copied over from tools?

yeah basically manually copying over the relevant changes and keeping the paths different since they are in different directories.

Reply:
> or keep it as is if it's matching the tools dir.
☝️